services/authenticator.js

```diff
@@ -61,12 +62,17 @@ function Authenticator() {
 
   this.registerAndCreateProject = (config) => {
     let guest = { email: config.email };
```

Comment:
`const guestParams`, it's confusing with the one after

Reply:
I just changed the `var` to `const` without touching the rest of the code.
Your suggestion is great but I'm afraid it will change a lot of codes and should be on another scheduled change IMO